cmd/kubermatic-api/options.go

```diff
@@ -91,9 +93,9 @@ func newServerRunOptions() (serverRunOptions, error) {
 	flag.StringVar(&s.presetsFile, "presets", "", "The optional file path for a file containing presets")
 	flag.StringVar(&s.swaggerFile, "swagger", "./cmd/kubermatic-api/swagger.json", "The swagger.json file path")
 	flag.StringVar(&rawAccessibleAddons, "accessible-addons", "", "Comma-separated list of user cluster addons to expose via the API")
+	flag.StringVar(&caBundleFile, "ca-bundle", "", "The path to the certificate for the CA that signed your identity provider’s web certificate.")
```

Comment:
Don't you think having the files in the Helm charts is enough? I never optimized our CLI for ease of use :-D (except of course for the installer), but this at least reminds me that this PR needs to adjust the hack/run-* scripts.

Reply:
There is very wonderful function, [x509.SystemCertPool()](https://pkg.go.dev/crypto/x509#SystemCertPool) which can deliver us "the default system CAs" in case when -ca-bundle is not provided.